python/cortex.py

```diff
@@ -105,7 +111,11 @@ def open(self):
         
         # As default, a Emotiv self-signed certificate is required.
         # If you don't want to use the certificate, please replace by the below line  by sslopt={"cert_reqs": ssl.CERT_NONE}
-        sslopt = {'ca_certs': "../certificates/rootCA.pem", "cert_reqs": ssl.CERT_REQUIRED}
+
+        file_dir_path = os.path.dirname(os.path.realpath(__file__))
```

Comment:
should use high level package to handle file path 
https://docs.python.org/3/library/pathlib.html